probe/endpoint/reporter.go

```diff
@@ -132,6 +135,11 @@ func (r *Reporter) addConnection(rpt *report.Report, localAddr, remoteAddr strin
 
 	// Update address topology
 	{
+		revRemoteAddr := ""
+		if r, err := r.revResolver.Get(remoteAddr); err != nil {
+			revRemoteAddr = r
+		}
+
```

Comment:
Scratch that. It's bad practice to put empty values in the nose metadata, so move this section down and make it:

```
name, err := r...get(...)
if err != nill {
    Remotenode = remotenode.
}
```